lang/ilias_de.lang

```diff
@@ -702,12 +702,13 @@ assessment#:#examid_in_test_res#:#ILIAS-Prüfungsnummer anzeigen
 assessment#:#examid_in_test_res_desc#:#In den Testergebnissen, der Statistik und dem Excel-Export wird die ILIAS-Prüfungsnummer dargestellt.
 assessment#:#exp_all_test_runs#:#Alle Durchläufe
 assessment#:#exp_best_pass#:#Bewerteter Durchlauf
-assessment#:#exp_eval_data#:#Evaluationsdaten exportieren als
+assessment#:#exp_eval_data#:#Evaluationsdaten exportieren
 assessment#:#exp_file_created#:#Exportdatei erzeugt.
+assessment#:#exp_grammar_as#:#als
 assessment#:#exp_scored_test_run#:#Bester Durchlauf
 assessment#:#exp_type_certificate#:#Zertifikat (PDF)
```

Comment:
In addition: 
This should read "_Bewerteter_ Durchlauf", analogous to the English "Scored test attempt" (thx to @Chris-Squirrel).